tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -56,20 +56,21 @@ Select Container Size
 Wait Until GPU Dropdown Exists
     [Documentation]    Verifies that the dropdown to select the no. of GPUs exists
     Wait Until Page Contains    Number of GPUs
-    Wait Until Page Contains Element    xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[2]
+    Page Should Not Contain    All GPUs are currently in use, try again later.
+    Wait Until Page Contains Element    xpath://button[contains(@aria-labelledby, "gpu-numbers")]
     ...    error=GPU selector is not present in JupyterHub Spawner
 
 Set Number Of Required GPUs
     [Documentation]  Sets the gpu count based on the ${gpus} argument
     [Arguments]  ${gpus}
-    Click Element  xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[2]
-    Click Element  xpath:${JUPYTERHUB_DROPDOWN_XPATH}\[2]/ul/li[.="${gpus}"]
+    Click Element  xpath://button[contains(@aria-labelledby, "gpu-numbers")]
```

Comment:
can you create a variable for the xpath? like it was done with  `JUPYTERHUB_DROPDOWN_XPATH`

Reply:
This will be addressed in another PR. @lugi0 ^

Reply:
#543